packages/jest-core/src/runJest.ts

```diff
@@ -19,6 +18,8 @@ import {Config} from '@jest/types';
 import {
   AggregatedResult,
   makeEmptyAggregatedTestResult,
+  formatTestResult,
+  formatTestResults,
 } from '@jest/test-result';
```

Comment:
Why weren't these being imported from `@jest/test-result` before?

Reply:
Missed it when refactoring. Will be good to trim out exports from jest-util for 25